### DISCUSSION TITLE
Validações do Domínio

### DISCUSSION THREAD

**Discussion by llmeireles** [balta-io/7196#39] [Perguntas e Respostas] [answered]
@andrebaltieri , na aula de manipulação e atualização de tarefa, podemos ter um problema ao buscar a tarefa na base antes de realizar o update do Título da mesma, nesse caso onde deveríamos fazer a validação desse resultado? Diretamente no Handler

**Comment by andrebaltieri** [comment, marked as answer]
@llmeireles pode deixar o código dentro de um Try/Catch, igual mostro no curso de Fundamentos do C#!

**Reply by llmeireles** [reply]
Justo @andrebaltieri,obrigado pelo retorno , mas no caso de não for uma exception por exemplo , se receber um ID de tarefa para atualização, onde a mesma não exista na base de dados, entendo ser uma validação de domínio e não uma exception, onde você indicaria para realizar esse tipo de validação?

**Reply by andrebaltieri** [reply]
Ai no caso você receberia um objeto nulo e poderia verificar isto e retornar um erro 404 para tela. Dá uma conferida no curso de ASP.NET 6, tem isto lá <3
